notebook/agentchat_agentops.ipynb

```diff
@@ -2,117 +2,142 @@
  "cells": [
   {
    "cell_type": "markdown",
+   "id": "a447802c88c8a240",
+   "metadata": {},
    "source": [
-    "# AgentOps Observability and Monitoring\n",
-    "Using [AgentOps](https://agentops.ai/) allows you to quickly and easily add agent observability and monitoring to your Autogen services."
-   ],
-   "metadata": {
-    "collapsed": false
-   },
-   "id": "5a3b20ae8f10e4e9"
+    "<img src=\"https://raw.githubusercontent.com/AgentOps-AI/agentops/main/docs/img/banner.png\" alt=\"Description of image\" width=\"60%\"/>\n",
+    "\n",
+    "[AgentOps](https://agentops.ai/?=autogen) provides session replays, metrics, and monitoring for AI agents.\n",
+    "\n",
+    "At a high level, AgentOps gives you the ability to monitor LLM calls, costs, latency, agent failures, multi-agent interactions, tool usage, session-wide statistics, and more. For more info, check out the [AgentOps Repo](https://github.com/AgentOps-AI/agentops).\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b354c068",
+   "metadata": {},
+   "source": [
+    "### Screenshots\n",
+    "<details open>\n",
+    "  <summary>Agent Dashboard</summary>\n",
+    "   <img src=\"https://raw.githubusercontent.com/AgentOps-AI/agentops/main/docs/img/overview.png\" style=\"width: 90%;\" alt=\"Agent Dashboard\"/>\n",
+    "</details>\n",
+    "\n",
+    "<details>\n",
+    "  <summary>Session Statistics</summary>\n",
+    "   <img src=\"https://raw.githubusercontent.com/AgentOps-AI/agentops/main/docs/img/session.png\" style=\"width: 90%;\" alt=\"Agent Dashboard\"/>\n",
+    "</details>\n",
+    "\n",
+    "<details>\n",
+    "  <summary>Session Replay</summary>\n",
+    "   <img src=\"https://raw.githubusercontent.com/AgentOps-AI/agentops/main/docs/img/replay.png\" style=\"width: 90%;\" alt=\"Agent Dashboard\"/>\n",
+    "</details>\n"
+   ]
   },
   {
    "cell_type": "markdown",
+   "id": "1731b1788d44533e",
+   "metadata": {},
    "source": [
-    "````{=mdx}\n",
-    ":::info Requirements\n",
```

Comment:
this formatting is intentional. there is a guide in the autogen docs on how to contribute additional notebooks and this is a required cell

Reply:
@bboynton97 Can you double check it it just looks weird in the notebook

Reply:
Yeah. looked very weird. Can easily revert tho

Reply:
yeah all the notebooks have that when you view directly but it allows them to render in their docs website